sql/src/main/java/io/crate/execution/dml/upsert/GeneratedColumns.java

```diff
@@ -52,11 +57,13 @@
 
     private final Map<Reference, Input<?>> toValidate;
     private final Map<Reference, Input<?>> toInject;
+    private final Map<Reference, Input<?>> maybeInjectNested;
```

Comment:
I think we could use clarification here that explains the difference between `toInject` and `maybeInjectNested`. 

The cases in which they apply, etc..